lib/adal.js

```diff
@@ -189,6 +200,48 @@ var AuthenticationContext = (function () {
         // callback from redirected page will receive fragment. It needs to call oauth2Callback
     };
 
+    AuthenticationContext.prototype.openPopup = function (urlNavigate, title, w, h) {
```

Comment:
Lets define this as a private method (even though it will still be on prototype). Also, please add comments to explain what this code is doing as it is assuming some variables to be defined in the current context.